demo/docbot/docbot.py

```diff
@@ -0,0 +1,64 @@
+import sys
+from tkinter.filedialog import Open
+from token import OP
+from unittest import result
```

Comment:
I think these imports may have been accidental.

Reply:
Yeah, they are. I was referencing another document for the imports and probably meant to remove them since they are not used. Thanks for the catch!